testing/ndctl/APKBUILD

```diff
@@ -1,42 +1,32 @@
 # Maintainer: Breno Leitao <breno.leitao@gmail.com>
 pkgname=ndctl
-pkgver=56
+pkgver=57.1
 pkgrel=0
 pkgdesc="Utility library for managing the libnvdimm (non-volatile memory device) sub-system in the Linux kernel"
 url="https://github.com/pmem/ndctl"
 arch="all"
 license="GPL"
 depends="libuuid json-c kmod"
 makedepends="autoconf automake libtool asciidoc xmlto kmod-dev eudev-dev util-linux-dev json-c-dev linux-headers"
-install=""
 subpackages="$pkgname-doc"
-source="$pkgname-$pkgver.tar.gz::https://github.com/pmem/ndctl/archive/v$pkgver.tar.gz
-0001-util-Enable-ndctl-to-compile-on-non-glibc-envs.patch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/pmem/$pkgname/archive/v$pkgver.tar.gz"
 
 builddir="$srcdir"/ndctl-$pkgver
 
 build() {
 	cd "$builddir"
-
-	./autogen.sh || return 1
-
+	./autogen.sh
```

Comment:
I manually did this upstream but just so you know for feature patch submission: Please invoke `./autogen.sh` in `prepare()` not in `build()`. Thanks!